docs/design/proposals/encrypted-pvc.md

```diff
@@ -0,0 +1,131 @@
+# Encrypted Persistent Volume Claims
+
+## Proposal
+
+Subject of this proposal is to add support for encryption of RBD volumes in
+Ceph-CSI.
+
+Some but not all the benefits of this approach:
+
+* guarantee encryption in transit to rbd without using messenger v2
```

Comment:
can you please explain what is this?

Reply:
so when you're using ceph to create volumes, you can have osd encryption that will ensure data is encrypted before it's stored on the disk, and with messenger v2 you can get encryption in transit to make sure data is encrypted while it's passed over the network (look [here](https://docs.ceph.com/docs/master/rados/configuration/msgr2/))

but encrypting network traffic is not required if you have encrypted data stored on rbd image itself, so it doesn't matter if you use msg v2 or not but you have guarantee that data is safe while it's transfered